edge/services/mms/horizon/service.definition.json

```diff
@@ -15,7 +15,7 @@
             "$SERVICE_NAME": {
                 "image": "${DOCKER_IMAGE_BASE}_$ARCH:$SERVICE_VERSION",
                 "privileged": false,
-		"binds": [ "/root/examples/edge/services/ss:/outside:rw" ]
+		"binds": [ "$PWD/ss:/outside:rw" ]
```

Comment:
If the user builds/publishes this service on their mac, and then runs it on a linux edge node, will the `$PWD` variable be a problem? Because `$PWD` will get evaluated at publish time, but the linux edge nodes likely have a different home directory. (It's possible i don't understand the full context here.)